fenix/app/src/main/res/values-night/colors.xml

```diff
@@ -12,6 +12,7 @@
     <color name="fx_mobile_layer_color_2">@color/photonDarkGrey30</color>
     <!-- Search -->
     <color name="fx_mobile_layer_color_3">@color/photonDarkGrey80</color>
+    <color name="fx_mobile_layer_color_search">@color/photonDarkGrey80</color>
```

Comment:
Could we move this below `fx_mobile_layer_color_info`? This will apply to the other colors.xml